splink/linker.py

```diff
@@ -2422,3 +2565,71 @@ def estimate_probability_two_random_records_match(
             " possible comparisons, we expect a total of around "
             f"{num_expected_matches:,.2f} matching pairs"
         )
+
+    def invalidate_cache(self):
+        """Invalidate the Splink cache.  Any previously-computed tables
+        will be recomputed.
+        This is useful, for example, if the input data tables have changed.
+        """
+        # Before Splink executes a SQL command, it checks the cache to see
+        # whether a table already exists with the name of the output table
+
+        # This function has the effect of changing the names of the output tables
+        # to include a different unique id
+
+        # As a result, any previously cached tables will not be found
+        self._cache_uid = ascii_uuid(8)
```

Comment:
as per comment above this changes the settings value, but doesn't propagate to the linker itself. With the additional effects of this method I don't think this can actually cause an issue elsewhere, but could do if one where to perhaps save/load settings including uuid for some reason.

Reply:
Very good spot - this is just an error, i will will fix it.